Source/WebKit/Shared/ShareableBitmap.cpp

```diff
@@ -109,8 +109,17 @@ RefPtr<ShareableBitmap> ShareableBitmap::createFromImageDraw(NativeImage& image)
         return nullptr;
 
     auto context = bitmap->createGraphicsContext();
-    if (!context)
-        return nullptr;
+    if (!context) {
+        // If createGraphicsContext() failed because the image colorSpace is not
+        // supported for output, fallback to SRGB.
+        bitmap = ShareableBitmap::create({ imageSize, DestinationColorSpace::SRGB() });
```

Comment:
Not sure if it's a big deal, but should we release the old `ShareableBitmap` before creating this new one, to avoid memory being allocated for both at the same time? Or, maybe we should move the fallback into `ShareableBitmap` somehow so it can re-use the same allocation?